metriq_gym/paths.py

```diff
@@ -0,0 +1,41 @@
+import os
+from pathlib import Path
+from importlib.metadata import distribution, PackageNotFoundError
+from platformdirs import user_data_dir, user_cache_dir
+
+AUTHOR_FALLBACK = "Unitary Foundation"
+
+
+def _load_app_meta(dist_name: str) -> tuple[str, str]:
+    try:
+        dist = distribution(dist_name)
+        meta = dist.metadata
+        name = meta["Name"] or dist_name
+        author = meta.get("Author", AUTHOR_FALLBACK)
+    except PackageNotFoundError:
+        name, author = dist_name, AUTHOR_FALLBACK
+    return name, author
+
+
+_APP_NAME, _APP_AUTHOR = _load_app_meta("metriq-gym")
+
+
+def get_data_db_path(filename: str = "localdb.jsonl") -> Path:
```

Comment:
Should these be private methods? If not, should we include docstrings and/or tests?

Reply:
done